packages/api/src/libs/osnap/utils.ts

```diff
@@ -0,0 +1,28 @@
+import assert from "assert";
+import { Json } from "../../types";
+import type { OsnapPluginData, Transaction } from "./interfaces";
+import { simulateTenderlyTx, TenderlySimulationParams } from "@uma/common";
```

Comment:
using the library reinis wrote in common, simplified code a lot, but needs to be updated for state overrides